src/numberGenerator.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const numberGenerator = () => {
+  const number = [];
+
+  while (number.length < 4) {
+    const digit = Math.floor(Math.random() * 9);
```

Comment:
Almost fixed, but now this code only generates digits in range 0-8.
Check out [Math.random docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random), `Math.random()` always generates numbers < 1, therefore `Math.random() * 9` will always be < 9.

Reply:
So, the `Math.floor` will never return 9, as your argument is always less than 9.

Reply:
Finally fixed ✅